apiserver/facades/agent/deployer/deployer_test.go

```diff
@@ -141,6 +136,17 @@ func (s *deployerSuite) SetUpTest(c *gc.C) {
 	)
 	c.Assert(err, jc.ErrorIsNil)
 	s.deployer = deployer
```

Comment:
You *could* put the deployer API setup inside `setupMocks` but since it is not really a mock, maybe it makes more sense to put it inside a separate function, `makeAPI` or something. Otherwise you could explicitly set it up inside every test case.

Reply:
That's a good idea. Will do it.